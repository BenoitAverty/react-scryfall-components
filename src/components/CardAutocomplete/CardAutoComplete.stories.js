import React from 'react';

import { storiesOf } from '@storybook/react';

import CardAutocomplete from '.';

storiesOf('CardAutocomplete', module).add('CardAutocomplete', () => (
  <CardAutocomplete />
));
