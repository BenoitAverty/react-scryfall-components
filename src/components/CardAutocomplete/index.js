import React from 'react'

export default () => <p>This is a card autocomplete</p>