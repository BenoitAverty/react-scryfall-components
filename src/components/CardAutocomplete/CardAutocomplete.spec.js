// Import all that is needed for testing
import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';

// Import the component under test
import CardAutocomplete from '.';

describe('CardAutocomplete component', () => {
  beforeEach(() => {
    cleanup();
  });

  test('snapshot', async () => {
    const { container } = render(<CardAutocomplete />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
