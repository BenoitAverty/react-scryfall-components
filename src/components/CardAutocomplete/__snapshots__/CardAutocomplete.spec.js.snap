// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardAutocomplete component snapshot 1`] = `
<p>
  This is a card autocomplete
</p>
`;
